tap_redshift/__init__.py

```diff
@@ -136,14 +141,7 @@ def discover_catalog(conn, db_schema):
             schema=schema,
             table=qualified_table_name,
```

Comment:
`database` (actually `database-name`) should be moved to `metadata`.
Also, is `table` the correct keyword arg here? The spec has `table-name` instead.

See: https://github.com/singer-io/getting-started/blob/master/docs/DISCOVERY_MODE.md#the-catalog

Reply:
`table` eventually translates to `table_name`. It needs to be passed as `table` in the `CatalogEntry` object then translates to `table_name` in the actual catalog result.